docs/source/get_started/new_project.md

```diff
@@ -56,24 +68,19 @@ repo_name: testing-kedro
 python_package: test_kedro
 ```
 
-To create this new project:
-
-```bash
-kedro new --config=<path>/config.yml
-```
-
-## Create a new project containing example code
 
-Use a [Kedro starter](../kedro_project_setup/starters.md) to create a project containing template code, to run as-is or to adapt and extend.
+## Create a new project containing starter code
 
-The following illustrates a project created with example code based on the familiar [Iris dataset](https://www.kaggle.com/uciml/iris).
+You can create a new Kedro project with a [starter](../starters/starters.md) that adds a set of code for a common project use case from the [range available](../starters/starters.md).
 
-The first step is to create the Kedro project using a starter to add the example code and data.
+The following illustrates a project created with example code for the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md). Navigate to your preferred directory and type the following:
 
 ```bash
-kedro new --starter=pandas-iris
+kedro new --starter=pandas-spaceflights
 ```
 
+After you enter `kedro new`, the command line interface will ask which tools you'd like to include, which will depend on the requirements of your project. The options are [described in more detail above](#tools-to-configure-the-new-project) and in the [documentation about the new project tools](../starters/new_project_tools.md).
```

Comment:
Iirc the tools selection workflow is not triggered when a starter is selected, will cross-check shortly

Reply:
Hmm, I thought that too but saw something in @merelcht's PR to suggest otherwise https://github.com/kedro-org/kedro-starters/pull/185/files

Reply:
@AhdraMeraliQB is correct, if you use the `--starter` flag you will not get the interactive menu to pick tools and you can't use `--starter` with `--tools/add-ons`. It's an either or workflow
- either you use --starter
- or you get to select tools through the menu or by using the flag --tools/add-ons (I'm not sure if it's renamed yet)

Reply:
Let me know if I need to change the text in the starters if it isn't clear there.

Reply:
Aha, OK thanks. Yes, I was definitely a bit misled by that text in the starters READMEs but I can change it. Thanks both.